travis/push-to-pypi.sh

```diff
@@ -33,9 +34,13 @@ DEV_TAG_NAME="acc_provision-${TRAVIS_TAG}.dev${TRAVIS_BUILD_NUMBER}"
 
 if [ -n "$PYPI_RELEASE" ] ; then
     #twine upload --repository-url https://pypi.org/legacy/ -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
+    # commented to test
+    #python setup.py sdist
+    #twine upload -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
+    #$SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}" "true"
     python setup.py sdist
-    twine upload -u ${PYPI_USER} -p ${PYPI_PASS} dist/$WHEEL_NAME
-    $SCRIPTS_DIR/push-to-cicd-status.sh "https://pypi.org/project/acc-provision/"${TRAVIS_TAG}"/#files" "${TAG_NAME}"
+    twine upload --repository testpypi -u ${TEST_PYPI_USER} -p ${TEST_PYPI_PASS} dist/$WHEEL_NAME
```

Comment:
we need to put it back to PYPI

Reply:
yes, please refer to comment: https://github.com/noironetworks/cicd/pull/10#discussion_r1321916140